main.go

```diff
@@ -310,6 +313,7 @@ type options struct {
 	labelKey               string
 	labelValue             string
 	allowOnlyReadyReplicas bool
+	annotatePodsOnChange   bool
```

Comment:
@matej-g I believe we would need an equivalent jsonnet change to add write permissions to the role to edit Pods for this?

Reply:
Hm that's a good catch, I don't remember if I had to do that by hand. I'll double check.